tests/e2e/helpers.bash

```diff
@@ -142,16 +144,16 @@ bria_init() {
     fi
   elif [[ "${wallet_type}" == "multisig" ]]; then
     local key1="tpubDE8HT914zGpxhJhgoMX35xgNyjHy5d1neGXHjTLAtuUssTA7tNWNs177JsFPbJwD5FBXCHJYbwUC9AzSEpYHC4hKgaCvZyZTuCbWfNUWXoM"
-    local key2="tpubDF2UP3xmRcJoi5LDYZKVoUj6MsaUrcPiTgV7Q2b8eWtnSsM3EhFeszEiG9iTU7Gixzym4a1F2T6gSp59y7iZrtTKZRAw6fqzbevphHxBzGz"
+    local key2="tpubDD4vFnWuTMEcZiaaZPgvzeGyMzWe6qHW8gALk5Md9kutDvtdDjYFwzauEFFRHgov8pAwup5jX88j5YFyiACsPf3pqn5hBjvuTLRAseaJ6b4"
     
     if [[ "${BRIA_CONFIG}" == "docker" ]]; then
       retry 10 1 bria_cmd import-xpub -x "${key1}" -n key1 -d m/48h/1h/0h/2h
-      bria_cmd import-xpub -x "${key2}" -n key2 -d m/48h/1h/0h/2h
-      bria_cmd create-wallet -n multisig sorted-multisig -x key1 key2 -t 1
+      bria_cmd import-xpub -x "${key2}" -n key2 -d m/84h/0h/0h
```

Comment:
I think the name is off here. You are importing `-n key2` but in the `create-wallet` call you are referencing `lnd_key`